src/transform/FooterLegal.js

```diff
@@ -29,13 +36,24 @@ const add = (content, licenseString, licenseSubstitutionString, containerID,
         ${licenseSubstitutionString}
       </a>
       ${licenseStringHalves[1]}
+      <br>
+      <div class="pagelib_footer_browser">
+      <a class='pagelib_footer_browser_link'>
```

Comment:
41-43 seem to be missing tag indentation which is inconsistent with the rest of this string..

Reply:
Did you mean indenting 42 (`${viewInBrowserString}`) or entire 41-43? 😃

Reply:
Line 31 is a div and it's children are indented. I was thinking this code should be consistent with that styles since it's all one block. :+1: